hack/lib/util.sh

```diff
@@ -485,9 +485,12 @@ kube::util::ensure_clean_working_dir() {
   while ! git diff HEAD --exit-code &>/dev/null; do
     echo -e "\nUnexpected dirty working directory:\n"
     git status -s | sed 's/^/  /'
```

Comment:
This line happens no matter what, which outputs `git status -s` twice in interactive terminals